### ISSUE TITLE
parse_latex converting `-1` to `-1*1`

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#24034]
### Discussed in https://github.com/sympy/sympy/discussions/24031

<div type='discussions-op-text'>

<sup>Originally posted by **gerryfletch** September  8, 2022</sup>
Hello!

I'm trying to decide whether a formula is in its simplest form, and a hurdle I'm running into is that formulas of the form `x - n` expands to `-1 * n` for any negative constant. Examples:

```python
parse_latex(-1) # -1       <- works as intended
parse_latex(3-1) # 3 - 1*1
parse_latex(3x-1) # 3*x - 1*1
parse_latex("-3 + x") # x - 3
```

I'm not sure why it's adding in `-1*`. I can't replicate the behaviour with string parsing, and it's only when the negative is on the RHS, as shown by the 4th example. 

If this isn't a bug/odd behaviour, what can I do to check if some expression parsed by latex is simplified?
This is problematic for me because `3x-1` is considered not simplified due to the rearrangement.</div>

**Comment by asmeurer** [comment]
```
>>> print(parse_latex('3x-1'))
3*x - 1*1
```

**Comment by asmeurer** [comment]
Ideally everything from parse_latex should be simplified. It shouldn't bypass evaluation, except maybe if an `evaluate=False` flag is passed, similar to `parse_expr`.

**Comment by oscargus** [comment]
Closed by #24096 (which linked the discussion).
